docs/source/user-guide/vision/data-classes/DetectionData.rst

```diff
@@ -86,48 +87,50 @@ Examples
             return imgs
 
         def batch_to_labels(self, batch):
-            """Convert a batch of labels to a tensor.
+            """Convert a batch bounding boxes to the required format.
 
             Parameters
             ----------
-            batch : torch.Tensor
-                The batch of labels to convert.
+            batch : torch.Tensor (N, B, 5)
```

Comment:
Technically in this example the back contains both image and label.
Also, is the original batch label a tensor? Given that B is different for each image?